CVLByExample/UnresolvedCallSummarization/README.md

```diff
@@ -0,0 +1,31 @@
+# Unresolved Call Summarization Example
+
+## Overview
+This repository contains a smart contract named `TrusterLenderPool`, featuring a `flashLoan` function. However, there exists a significant vulnerability within this function. The vulnerability allows the pool contract to act as the `msg.sender` for any call, presenting a critical security flaw. This example showcases the utilization of a new feature in CVL called Unresolved Call Summarization. This feature accentuates how the vulnerability can be exploited to grant the attacker an infinite allowance, subsequently leading to the potential draining of all funds within the contract.
+
+## Usage
+
+```cvl
+function _._ external => DISPATCH [
+    token.approve(address, uint256), 
+    currentContract._
+] default NONDET;
+```
+In this implementation, we summarize all unresolved calls into one of the following categories:
+- If the signature call matches the `approve` function, it is categorized as `token.approve(address, uint256)`.
+- If the signature call matches any of the functions within `currentContract`, their implementations are utilized.
+- Otherwise, it triggers a `NONDET` (nondeterministic) summary.
+
+## Executions
+
+1. **Certora Run Command**
+    - Command:
+        ```bash
+        certoraRun truster.conf --server production --prover_version master
```

Comment:
no need --server production --prover_version master we will post when it is in production

Reply:
Done